native/SampleApps/MobileSyncExplorer/MobileSyncExplorer/bootconfig.plist

```diff
@@ -8,8 +8,8 @@
 		<string>api</string>
 	</array>
 	<key>oauthRedirectURI</key>
-	<string>testsfdc:///mobilesdk/detect/oauth/done</string>
+	<string>https://msdk-enhanced-dev-ed.my.site.com/services/oauth2/echo</string>
 	<key>remoteAccessConsumerKey</key>
-	<string>3MVG98dostKihXN53TYStBIiS8FC2a3tE3XhGId0hQ37iQjF0xe4fxMSb2mFaWZn9e3GiLs1q67TNlyRji.Xw</string>
+	<string>3MVG9CEn_O3jvv0wTqRT0Le6tmzX.EQ9ZvtHL1TG3gHFV.4IvKZyXw5SgdiVPi61mXrpu40mCOhKevEfYNMOm</string>
```

Comment:
This is not necessary for Native Login (and in fact will prevent fallback to web based auth).